### DISCUSSION TITLE
docker搭建的配置了bingmaps的api但是网站没有打开地图的开关

### DISCUSSION THREAD

**Discussion by shmilyabyy** [jason5ng32/MyIP#209] [Q&A] [answered]
![image](https://github.com/jason5ng32/MyIP/assets/77541484/f53cc72c-2968-484a-929e-6681948ca724)
只有以上三个。

**Comment by jason5ng32** [comment, marked as answer]
是 Bug，已经修复。

**Reply by shmilyabyy** [reply]
还是没有解决，我的还是没有地图开关。
日志报错没了，但是还是没有地图。

**Reply by shmilyabyy** [reply]
已解决
